examples/basics/gloo/spatial_filters.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# vispy: gallery 2
+"""
+Example demonstrating spatial filtering using spatial-filters fragment shader.
+Left and Right Arrow Keys toggle through available filters.
+"""
+
+import os
+import numpy as np
+
+from vispy.util.transforms import ortho
+from vispy.io import _data_dir
+from vispy import gloo
+from vispy import app
+
+
+# Image to be displayed
+W, H = 100, 100
+I = np.random.uniform(0,1,(5,5,3)).astype(np.float32)
```

Comment:
Maybe a non random image would be better suited to show actual interpolation.  Your central point in a 5x5 was fine.